app/Helpers/database/achievement.php

```diff
@@ -27,13 +27,15 @@ function getAchievementsList(
         'limit' => $limit,
     ];
 
-    $innerJoin = "";
+    $innerJoinPlayerAchievements = "";
     $withAwardedDate = "";
     if ($params > 0 && $user) {
-        $innerJoin = "LEFT JOIN player_achievements AS pa ON pa.achievement_id = ach.ID AND pa.user_id = " . $user->id;
+        $innerJoinPlayerAchievements = "LEFT JOIN player_achievements AS pa ON pa.achievement_id = ach.ID AND pa.user_id = " . $user->id;
         $withAwardedDate = ", COALESCE(pa.unlocked_hardcore_at, pa.unlocked_at) AS AwardedDate";
     }
 
+    $innerJoinUserAccounts = "LEFT JOIN UserAccounts AS ua ON ua.ID = ach.user_id";
```

Comment:
Why use a variable for this? There's other `INNER JOIN`s directly in the query.

Reply:
Good catch. This is leftover cruft from refactoring work. Fixed in latest.